app/templates/iat-search-results.html

```diff
@@ -44,6 +44,7 @@ <h1>{{ page_title_value }}</h1>
                             'previous_area': previous_area,
                             'previous_ons_id': previous_ons_id,
                             'previous_surname_filter': previous_surname_filter,
+			    'previous_employee_id': previous_employee_id,
```

Comment:
The indentation here seems to be off! It may have inserted tabs instead of spaces.